src/mainwindow.h

```diff
@@ -188,8 +190,14 @@ class MainWindow : public QMainWindow {
 
     void resetBrokenTagNotesLinkFlag();
 
+    Q_INVOKABLE QString getWorkspaceUuuid(const QString &workspaceName);
```

Comment:
We'll need documentation for all those on https://www.qownnotes.org/scripting/classes.html#mainwindow

Reply:
see: https://github.com/pbek/QOwnNotes/tree/develop/webpage/src/scripting

Reply:
Updated docs.

Reply:
Where?

Reply:
Did you push your documentation commit? I can't see any. 😁

Reply:
Something must have happened with that change. Will redo it :D

Reply:
Forgot to create the PR :)

Reply:
`getWorkspaceUuuid` has one `u` to many, hasn't it?

Reply:
I'll refactor that.